kernel/basic.ml

```diff
@@ -148,6 +148,7 @@ let rev_mapi f l =
   in
   rmap_f 0 [] l
 
+let concat l1 = function [] -> l1 | l2 -> l1@l2
```

Comment:
Maybe you should add a hint to say that this function should be inlined (with [@@ inline]

Reply:
List.append exists in the standard library. Is it a good idea to create (almost) an alias?
Is it really more efficient to add a test?